pkg/kv/postgres/store.go

```diff
@@ -42,37 +41,15 @@ const (
 
 	// DefaultPartitions Changing the below value means repartitioning and probably a migration.
 	// Change it only if you really know what you're doing.
-	DefaultPartitions = 100
-
-	DefaultMaxOpenConnections    = 25
-	DefaultMaxIdleConnections    = 25
-	DefaultConnectionMaxLifetime = 5 * time.Minute
-	DefaultScanPageSize          = 1000
+	DefaultPartitions   = 100
+	DefaultScanPageSize = 1000
```

Comment:
Are these defaults or constants? Didn't see where they could be configured

Reply:
DefaultPartitions also didn't found a way to configure it. The value used as constant. 
Currently the DefaultScanPageSize used by default - without any option to change, just from a test code.